src/view/com/util/post-embeds/VideoEmbedInner/VideoEmbedInnerWeb.tsx

```diff
@@ -33,47 +34,57 @@ export function VideoEmbedInnerWeb({
   const hlsRef = useRef<Hls | undefined>(undefined)
 
   useEffect(() => {
-    if (!ref.current) return
-    if (!Hls.isSupported()) throw new HLSUnsupportedError()
+    let hls: Hls | undefined
+    // @ts-expect-error - dynamic import
+    import('hls.js/dist/hls.min').then(
```

Comment:
i actually think we want to kick it off early — say in the module scope — as long as it’s non blocking. since otherwise we’re delaying when the first video can be shown. 


consider playing with these: https://webpack.js.org/guides/code-splitting/#prefetchingpreloading-modules

Reply:
basically it shouldn’t hold up the rest of the bundle but it also shouldn’t be deferred until render. you want to kick it off early on

Reply:
Just writing down so I don't forget:

Magic comment seems to not really be working. Better option might be to try and put this in a separate chunk which can get loaded in parallel maybe inside the webpack config. Docs are pretty mid unfortunately so need to play with this